Source/WebKit/WebProcess/WebPage/wc/WCUpdateInfo.serialization.in

```diff
@@ -39,4 +39,65 @@ struct WebKit::WCUpdateInfo {
     Vector<WebKit::WCLayerUpdateInfo> changedLayers;
 }
 
+[Nested,OptionSet] enum class WebKit::WCLayerChange : uint8_t {
```

Comment:
In the code it's defined as: `enum class WCLayerChange : uint32_t {`. This needs to be consistent

Reply:
This should be uint32_t